docs/database/ftl.md

```diff
@@ -93,22 +82,22 @@ ID | Query Type
 <!-- Query | A | AAAA | ANY | SRV | SOA | PTR | TXT -->
 
 ### Supported status types
-ID | Query Type
---- | ---
-0 | Unknown status (was not answered by forward destination)
-1 | Blocked by `gravity.list`
-2 | Permitted + forwarded
-3 | Permitted + replied to from cache
-4 | Blocked by wildcard
-5 | Blocked by `black.list`
-6 | Blocked by upstream server (known blocking page IP address)
-7 | Blocked by upstream server (`0.0.0.0` or `::`)
-8 | Blocked by upstream server (`NXDOMAIN` with `RA` bit unset)
-
-### Example for interaction with the FTL long-term database
+ID | Status | | Details
+--- | --- | --- | ---
+0 | Unknown status | &#x2754; | (was not answered by forward destination)
```

Comment:
This can just be "Unknown" instead of "Unknown status", and the parentheses can be removed to keep consistent with the rest of the statuses.